### DISCUSSION TITLE
How do we load reference data such as HPO onto our local instance of seqr?

### DISCUSSION THREAD

**Discussion by Mullai** [broadinstitute/seqr#2306] [Q&A] [answered]
Hi,

We are pilot testing a local install of seqr at the Baylor College of Medicine's genome center. We were able to successfully setup a seqr instance on AWS. Next, we were planning to add reference data such as HPO to our local install of seqr but are unsure of the procedure to do so. Appreciate your help on this.

Thank you,
Mullai

**Comment by hanars** [comment, marked as answer]
If you are using our recommended docker image, all reference data should already be installed. You can manually add the reference data by running 
```
python -u manage.py update_all_reference_data --use-cached-omim
```
However, that is run in the docker entrypoint, so it you probably want to read through your logs and figure out why that wasn't successfully run and make sure that all the other set up commands ran successfully

**Reply by Mullai** [reply]
Thank you! we were able to successfully resolve this issue.
